### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by mkempf**
Some gamelogic and styling

### CODE REVIEW

File: src/main/java/ch/elca/students/sudokubattleroyal2/game/SudokuGenerator.java [hunk 1]
```diff
@@ -22,11 +22,19 @@
...
+        return makeHoles(difficulty,game);
     }
 
 
-    /**
+    private static int[][] copyArray(int[][] array) {
+    	int[][] copy = Arrays.stream(array)
+                .map((int[] row) -> row.clone())
+                .toArray((int length) -> new int[length][]);
+    	return copy;
```
Comment (ret0) [inline src/main/java/ch/elca/students/sudokubattleroyal2/game/SudokuGenerator.java]: inline

File: src/main/resources/static/index.html [hunk 2]
```diff
@@ -2,35 +2,55 @@
...
+			<h4 class="form-signin-heading">Welcome to Sudoku Battle Royal 2</h4>
+			<label for="login-name" class="sr-only">Email address</label>
+			<input type="email" id="login-name" name="username" class="form-control" placeholder="Enter your email address" required autofocus>
+			<label for="inputPassword" class="sr-only">Password</label>
+			<input type="password" id="inputPassword" name="password" class="form-control" placeholder="Password">
+			<input class="btn btn-primary btn-lg btn-block" type="submit" value="Log In"/>
+		</form>
+	
+	
+	    <!-- <div class="login">
```
Comment (ret0) [inline src/main/resources/static/index.html]: delete?

File: src/main/java/ch/elca/students/sudokubattleroyal2/game/SudokuGenerator.java (hunk 1 above)
Comment (mkempf) [inline src/main/java/ch/elca/students/sudokubattleroyal2/game/SudokuGenerator.java]: done

File: src/main/resources/static/index.html (hunk 2 above)
Comment (mkempf) [inline src/main/resources/static/index.html]: done

**Pull request merged**